distribution/sdk/artemis-sandbox/pom.xml

```diff
@@ -50,6 +50,11 @@
             <version>2.18.0</version>
         </dependency>
         <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-stream</artifactId>
+            <version>2.18.0</version>
```

Comment:
The SDK is currently not hooked up to the main pom, I have to hardcode the version to 2.18.3